real_estate/models/estate_property.py

```diff
@@ -0,0 +1,143 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+import datetime
+
+class EstateProperty(models.Model):
+
+    _name = 'estate.property',
+    _description = 'Real Estate Property'
+
+    name = fields.Char(
+        string='Property Name', 
+        required=True, 
+        help="Enter the name of the property"
+    )
+
+    description = fields.Text(
```

Comment:
```suggestion
    )
    description = fields.Text(
```
We can remove this extra line, we only add a extra lines when we want to clarify the use of some few fields are different

Reply:
done sir! I remove extra lines